code/run.py

```diff
@@ -0,0 +1,46 @@
+#! /usr/bin/env python3
+import os
+
+# ---------------------------------------------------------------------------
+# Original images from http://mrl.nyu.edu/~ajsecord/
+# -> http://cs.nyu.edu/~ajsecord/npar2002/StipplingOriginals.zip
+# ---------------------------------------------------------------------------
+
+os.system("./stippler.py ../data/original/plant4h.png --save \
+           --n_point 20000 --n_iter 50 --pointsize 1.5 1.5 --figsize 6 \
+           --threshold 255 --force --interactive")
+
+os.system("./stippler.py ../data/original/plant5_700x700.png  --save
+           --n_point 20000 --n_iter 50 --pointsize 2.5 2.5 --figsize 8 \
+           --threshold 220 --force --interactive")
+
+os.system("./stippler.py ../data/original/plant2_400x400.png  --save \
```

Comment:
Oh, but that also does not work because file associations. Forget the above and do:
```py
import sys
cmd = sys.executable

os.system(cmd + " ./stippler.py ../data/original/plant4h.png --save \
...
```